utils/src/field_helpers.rs

```diff
@@ -122,26 +122,27 @@ pub trait FieldHelpers<F> {
     where
         F: PrimeField,
     {
-        F::size_in_bits() / 8 + (F::size_in_bits() % 8 != 0) as usize
+        (F::MODULUS_BIT_SIZE / 8) as usize + (F::MODULUS_BIT_SIZE % 8 != 0) as usize
     }
 
     /// Get the modulus as `BigUint`
     fn modulus_biguint() -> BigUint
     where
         F: PrimeField,
     {
-        BigUint::from_bytes_le(&F::Params::MODULUS.to_bytes_le())
+        BigUint::from_bytes_le(&F::MODULUS.to_bytes_le())
     }
 }
 
 impl<F: Field> FieldHelpers<F> for F {
     fn from_bytes(bytes: &[u8]) -> Result<F> {
-        F::deserialize(&mut &*bytes).map_err(|_| FieldHelpersError::DeserializeBytes)
+        F::deserialize_uncompressed(&mut &*bytes).map_err(|_| FieldHelpersError::DeserializeBytes)
     }
 
     fn from_hex(hex: &str) -> Result<F> {
         let bytes: Vec<u8> = hex::decode(hex).map_err(|_| FieldHelpersError::DecodeHex)?;
-        F::deserialize(&mut &bytes[..]).map_err(|_| FieldHelpersError::DeserializeBytes)
+        F::deserialize_uncompressed(&mut &bytes[..])
```

Comment:
As always, I would be happy to have some regression tests. One test could be checking that deserialization/serialization on Fp::MODULUS + 1 works/doesn't work on both version. Same for the serialization. And also have some PBT (serialization followed by deserialization gives the same value).